index.html

```diff
@@ -45,8 +46,9 @@ <h2>You´re in good company</h2>
     </div>
   </main>
   <footer>
-    <div>
+    <div class="info1">
```

Comment:
try to name your css relevant to the elements function. watch your indentation here :)